packages/core/helper-plugin/src/features/AppInfo.tsx

```diff
@@ -1,44 +1,35 @@
-import * as React from 'react';
+import { ReactNode, createContext, useContext, useMemo } from 'react';
```

Comment:
Could we keep the import like this import * as React from 'react'; and reference everything on the React namespace? React.ReactNode, React.createContext.

It's a convention we are establishing but don't have a lint rule setup yet.

Reply:
Reverted, I'll keep that in mind in the future.